nixos/modules/services/networking/nftables.nix

```diff
@@ -0,0 +1,173 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  cfg = config.networking.nftables;
+in
+{
+  ###### interface
+  
+  options = {
+    networking.nftables.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description =
+        ''
+          Whether to enable nftables.  nftables is a Linux-based packet
+          filtering framework intended to replace frameworks like iptables.
+          
+          This conflicts with the standard networking firewall, so make sure to
+          disable it before using nftables.
+        '';
+    };
+    networking.nftables.ruleset = mkOption {
+      type = types.lines;
+      default =
+        ''
+          table inet filter {
+            # Block all IPv4/IPv6 input traffic except SSH.
```

Comment:
```
ct state {established, related} accept
iifname lo accept
```
> Block all IPv4/IPv6 input traffic except SSH.

Doesn't block all traffic... (only new traffic)
I don't like that port 22 is hard-coded.